explorer/utils/dockerConnectionError.ts

```diff
@@ -37,8 +37,8 @@ export function showDockerConnectionError(actionContext: IActionContext, error:
     actionContext.suppressErrorDisplay = true;
     vscode.window.showErrorMessage(parseError(wrappedError).message, ...items).then(response => {
         if (response) {
-            // tslint:disable-next-line:no-unsafe-any
-            opn(response.url);
+            // tslint:disable-next-line:no-floating-promises
+            openExternal(response.url);
```

Comment:
How come some places you added an `await` and some places you just disable the lint error? I think in most cases it should be safe to await openExternal. (this was not true of `opn` because awaiting `opn` would never return as far as I could tell)

Reply:
Fair enough - I think awaiting `openExternal` is technically cleaner, but realistically I don't think it matters